patches/01-fix_build.diff

```diff
@@ -0,0 +1,51 @@
+diff --git a/data/appdata/meson.build b/data/appdata/meson.build
```

Comment:
Please add a comment section at the top saying what this patch is for.

Reply:
Done.